tests/test_estimator.py

```diff
@@ -13,8 +12,10 @@
     "RFC",
     "XGBC",
     "PLSC",
-    SVC(random_state=1),
-    Pipeline([("norm", Normalizer()), ("model", SVC(random_state=1))]),
+    SVC(kernel="linear", random_state=1),
+    Pipeline([("norm", Normalizer()), ("model", SVC(kernel="linear", random_state=1))]),
+    LinearRegression(),
+    LogisticRegression(),
```

Comment:
Maybe adding couple more regressor cases? so far it's just one.

Reply:
sure